difftest/libspike_interfaces/spike_interfaces_c.h

```diff
@@ -22,10 +22,10 @@ spike_state_t* proc_get_state(spike_processor_t* proc);
 
```

Comment:
Please add documentation to header files.

Reply:
And please think a question:
if we are going to use sail/qemu to replace spike in the future, how do we decouple this?
maybe `riscv_architecture_events.h`